src/components/project/models.ts

```diff
@@ -0,0 +1,17 @@
+export interface Text {
```

Comment:
Lets name this IText so that we can keep whats an interface and what is not an interface straight. Thas also a convention in TypeScript.